src/coreclr/nativeaot/Bootstrap/main.cpp

```diff
@@ -2,7 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 #include <stdint.h>
+
+#ifdef HAS_ADDRESS_SANITIZER
 #include "minipal/utils.h"
```

Comment:
If asansupport.cpp was the only reason for this include, then it should include this header (it already has a guard).

Reply:
Good catch. Looks like it already includes it so the line is unnecessary as-is.